build.sbt

```diff
@@ -23,14 +23,15 @@ resolvers += sonatypeSnapshot
 scalaVersion := "2.13.1"
 crossScalaVersions := Seq(scalaVersion.value, "2.12.10")
 
-libraryDependencies += "com.typesafe.akka" %% "akka-remote" % akkaVersion
+libraryDependencies += "com.typesafe.akka" %% "akka-actor" % akkaVersion
 libraryDependencies += "com.esotericsoftware" % "kryo-shaded" % "4.0.2"
 libraryDependencies += "org.lz4" % "lz4-java" % "1.6.0"
+libraryDependencies += "org.agrona" % "agrona" % "0.9.31" // should match akka-remote/aeron inherited version
```

Comment:
more correct would be when this version would be defined with the akka version but only a real problem if this is the higher version ->ok as long as we match this to akka 2.5 and not akka 2.6